applications/make_regular_arrays.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+
+    This application creates the layout array files (ECSV) of regular arrays
+    with one telescope at the center of the array and with 4 telescopes
+    in a square grid. These arrays are used for trigger rate simulations.
+
+    The array layout files created should be available at the data/layout directory.
+
+    Command line arguments
+    ----------------------
+    site_pars (str, optional)
+        Site parameters file in yaml format. If not given, the default one
+        from data/layout will be used.
+    verbosity (str, optional)
+        Log level to print (default=INFO).
+
+    Example
+    -------
+    Runtime < 1 min.
+
+    python applications/make_regular_arrays.py
+'''
+
+import logging
+import argparse
+
+import astropy.units as u
+from astropy.io.misc import yaml
+
+import simtools.io_handler as io
+import simtools.util.general as gen
+from simtools.layout.layout_array import LayoutArray
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(
+        description=(
+            'Calculate and plot the PSF and eff. mirror area as a function of off-axis angle '
+            'of the telescope requested.'
+        )
+    )
+    parser.add_argument(
+        '--site_pars',
+        type=str,
+        help=(
+            'Site parameters file in yaml format. If not given, the '
+            'default one from data/layout will be used.'
+        )
+    )
+    parser.add_argument(
+        '-v',
+        '--verbosity',
+        dest='logLevel',
+        action='store',
+        default='info',
+        help='Log level to print (default is INFO)'
+    )
+
+    args = parser.parse_args()
+    label = 'make_regular_arrays'
+
+    logger = logging.getLogger(label)
+    logger.setLevel(gen.getLogLevelFromUser(args.logLevel))
+
+    # Getting site pars file
+    if args.site_pars is not None:
+        siteParsFile = args.site_pars
+        logger.debug('Reading site parameters from {}'.format(siteParsFile))
+    else:
+        siteParsFile = io.getDataFile('layout', 'site_parameters.yml')
+        logger.debug('Reading default site parameters from {}'.format(siteParsFile))
+
+    try:
+        with open(siteParsFile, 'r') as f:
+            sitePars = yaml.load(f)
+    except FileNotFoundError:
+        msg = 'Site parameter file ({}) could not be opened/read'.format(siteParsFile)
+        logger.error(msg)
+        raise FileNotFoundError(msg)
+
+    # Telescope distances for 4 tel square arrays
+    # !HARDCODED
+    telescopeDistance = {'LST': 57.5 * u.m, 'MST': 70 * u.m, 'SST': 80 * u.m}
```

Comment:
@RaulRPrado Why is this needed? Is this for tests? So you define telescope distances for square layouts here. The distances are very small.

Reply:
These are the square arrays that we use for trigger rate simulations.
It's very specific and it will only be used in this application. And still, it's an application that is used only once.

Reply:
OK - and we only do single-telescope trigger there, no array trigger, correct? I am just surprised by the small telescope distances.

Reply:
For LST, the standard is stereo trigger with square array. For MST and SST we use single telescope.
But these are the standards, which doesn't mean we can't have the option of building square array, for whatever reason we need.
I will double check the distances.

Reply:
In sim_telarray there are CORSIKA input files for square arrays for all telescope tyles.
You can find, for example, INPUTS_CTA-trg-test-prod3-MST4_proton. 
The distances were taken from there. They might be outdated since they are from prod3.

Reply:
Couldn't we have that as a configuration in the DB - trigger array layouts?

Reply:
Yes, I think we can add it into the sites collection.